### PR TITLE
Create Printer Health Check

### PR DESCRIPTION
**Pull request by Annabel1054**

### CODE REVIEW

File: proto/print.proto [hunk 1]
```diff
@@ -3,15 +3,24 @@ syntax = "proto3";
...
+    string message = 1;
+}
+
+message HealthCheckUpRequest {
+    string member_name = 1;
+}
+
+message PrinterHealthStatus {
     string message = 1;
 }
```
Comment (evanugarte) [inline proto/print.proto:26]: add a blank new line at the bottom of this file

Comment (jerrylee17) [inline proto/print.proto:26]: New line

File: client/printing/print_pb.js [hunk 2]
```diff
@@ -404,4 +406,288 @@ proto.PrintResponse.prototype.setMessage = function(value) {
...
+ * @suppress {unusedLocalVariables} f is only used for nested messages
+ */
+proto.HealthCheckUpRequest.serializeBinaryToWriter = function(message, writer) {
+  var f = undefined;
+  f = message.getMemberName();
+  if (f.length > 0) {
+    writer.writeString(
+      1,
+      f
+    );
```
Comment (hsorathia) [inline client/printing/print_pb.js:531]: should this be in 4 lines?

File: client/printing/print_pb.js [hunk 3]
```diff
@@ -404,4 +406,288 @@ proto.PrintResponse.prototype.setMessage = function(value) {
...
+ * Static version of the {@see toObject} method.
+ * @param {boolean|undefined} includeInstance Whether to include the JSPB
+ *     instance for transitional soy proto support:
+ *     http://goto/soy-param-migration
+ * @param {!proto.PrinterHealthStatus} msg The msg instance to transform.
+ * @return {!Object}
+ * @suppress {unusedLocalVariables} f is only used for nested messages
+ */
+proto.PrinterHealthStatus.toObject = function(includeInstance, msg) {
+  var f, obj = {
```
Comment (hsorathia) [inline client/printing/print_pb.js:597]: use `let` instead of `var`

File: client/printing/print_pb.js (hunk 2 above)
Comment (evanugarte) [reply client/printing/print_pb.js:531]: lmaooo this is auto generated!! we can change this

File: client/printing/print_pb.js (hunk 3 above)
Comment (evanugarte) [reply client/printing/print_pb.js:597]: this is google generated stuff, not us

**Pull request merged**